core-common/src/main/java/org/zalando/nakadi/domain/EventTypeBase.java

```diff
@@ -74,6 +74,10 @@
     @JsonInclude(JsonInclude.Include.NON_NULL)
     private Audience audience;
 
+    @Nullable
+    @JsonInclude(JsonInclude.Include.NON_NULL)
+    private EventAuthField eventAuthField;
```

Comment:
You forgot `@Valid` annotation. Without it validation of `EventAuthField` won't be triggered